goodtablesio/plugins/github/tasks.py

```diff
@@ -67,3 +69,8 @@ def _get_job_files(top):
         for file_name in file_list:
             out.append(file_name)
     return out
+
+
+def _remove_repo(clone_url, clone_dir):
+    log.info('Removing repo {0} from {1}'.format(clone_url, clone_dir))
```

Comment:
This seems more like debug. And we can simplify it to `Remove cloned repo for {0}'.format(clone_url)`